tests/integration/pox/stacking/direct-stacking-good-order.spec.ts

```diff
@@ -98,25 +91,31 @@ describe("testing solo stacker increase without bug", () => {
 
     // Asserts about pox info for better knowledge sharing
     expect(poxInfo.contract_id).toBe("ST000000000000000000002AMW42H.pox-2");
-    expect(poxInfo.pox_activation_threshold_ustx).toBe(50_286_942_145_278);
+    // expect(poxInfo.pox_activation_threshold_ustx).toBe(50_286_942_145_278);
     expect(poxInfo.current_cycle.id).toBe(1);
-    expect(poxInfo.current_cycle.min_threshold_ustx).toBe(20_960_000_000_000);
+    // expect(poxInfo.current_cycle.min_threshold_ustx).toBe(20_960_000_000_000);
 
     // Assert that the next cycle has 100m STX locked
     expect(poxInfo.current_cycle.stacked_ustx).toBe(0);
     expect(poxInfo.current_cycle.is_pox_active).toBe(false);
     expect(poxInfo.next_cycle.stacked_ustx).toBe(100_000_000_000_111);
 
-    const poxAddrInfo0 = await readRewardCyclePoxAddressList(network, 2, 0);
-    const poxAddrInfo0CV = hexToCV(poxAddrInfo0.data) as SomeCV<TupleCV>;
-    const poxAddrInfoStr0 = cvToString(hexToCV(poxAddrInfo0.data));
+    const poxAddrInfo0 = (await readRewardCyclePoxAddressForAddress(
+      network,
+      2,
+      Accounts.WALLET_2.stxAddress
+    )) as Record<string, ClarityValue>;
     // There is no bug here because total stack was equal to Bob's stacked amount when Bob called stack-increase.
-    expect(cvToString(poxAddrInfo0CV.value.data["total-ustx"])).toBe(
-      "u50000000000110"
-    );
+    expect(cvToString(poxAddrInfo0["total-ustx"])).toBe("u50000000000110");
 
-    const poxAddrInfo1 = await readRewardCyclePoxAddressList(network, 2, 1);
-    const poxAddrInfoStr1 = cvToString(hexToCV(poxAddrInfo1.data));
-    expect(poxAddrInfoStr1).toContain("(total-ustx u50000000000001)");
+    // There is no bug here because total stack was 0 when stack-increase was called.
+    expect(poxAddrInfo0["total-ustx"]).toEqual(uintCV(50000000000110));
+
+    const poxAddrInfo1 = (await readRewardCyclePoxAddressForAddress(
+      network,
+      2,
+      Accounts.WALLET_1.stxAddress
+    )) as Record<string, ClarityValue>;
+    expect(poxAddrInfo1["total-ustx"]).toEqual(uintCV(50000000000001));
```

Comment:
I like this style of assertion

Reply:
I agree. I'm quickly updating all the others to match this.